python/app.py

```diff
@@ -0,0 +1,25 @@
+import jwt
+import time
+import os
+from urllib.parse import urljoin
+from flask import Flask
```

Comment:
Generally, it's preferred to separate the built-in imports vs 3rd party, so it's clear, like so: 

```
import os
import time
from urllib.parse import urljoin

from flask import Flask
import jwt
```